docker/Dockerfile

```diff
@@ -82,7 +82,8 @@ ENV DEBIAN_FRONTEND=noninteractive
 ARG GET_PIP_URL
 
 # Install system dependencies and uv, then create Python virtual environment
-RUN echo 'tzdata tzdata/Areas select America' | debconf-set-selections \
+RUN --mount=type=cache,id=apt-build,target=/var/lib/apt/lists \
```

Comment:
`base` uses `id=apt-build`, and `vllm-base` uses `id=apt-final`, since they are based on different ubuntu versions.